src/SFA.DAS.EmployerFinance.Web/Filters/GoogleAnalyticsViewBagFilter.cs

```diff
@@ -0,0 +1,23 @@
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+using Microsoft.Extensions.DependencyInjection;
+using SFA.DAS.EmployerFinance.Configuration;
+using StructureMap;
+
+namespace SFA.DAS.EmployerFinance.Web.Filters
+{
+    public class GoogleAnalyticsViewBagFilter : ActionFilterAttribute
+    {
+        public override void OnActionExecuting(ActionExecutingContext context)
+        {
+            if (!(context.Controller is Controller controller))
+            {
+                return;
+            }
+            
+            var container = context.HttpContext.RequestServices.GetService<IContainer>();
```

Comment:
Try to avoid taking a dependency on `IContainer`, you should be able to resolve `GoogleAnalyticsConfiguration` directly here instead.

Reply:
Filter has now been removed